tests/test_views.py

```diff
@@ -2810,6 +2810,18 @@ def test_get_pointed_private(self):
         assert_equal(pointed[0]['title'], 'Private Component')
         assert_equal(pointed[0]['authorShort'], 'Private Author(s)')
 
+    def test_can_template_project_linked_to_each_other(self):
+        project2 = ProjectFactory(creator=self.user)
+        self.project.add_pointer(project2, auth=Auth(user=self.user))
+        self.project.save()
```

Comment:
minor: it looks like `add_pointer` calls `_add_pointers` which has a `node.save` if successful, so I think these saves might be redundant